.github/dependabot.yml

```diff
@@ -84,9 +84,9 @@ updates:
       timezone: "Asia/Tokyo"
 
   - package-ecosystem: "github-actions"
-    target-branch: "3.13"
+    target-branch: "3.14"
     groups:
-      actions on branch 3.13:
+      actions on branch 3.14:
```

Comment:
@josh-wong 
It's off topic, but I think we should highlight versions that no longer provide maintenance support (i.e., 3.13 as of now) differently. (a little lighter than the unsupported versions?)
https://scalardb.scalar-labs.com/docs/latest/releases/release-support-policy